src/settings.c

```diff
@@ -11,7 +11,7 @@
 #endif
 
 #include "rules.h" // put before config.h to fix missing include
-#include "config.h"
+#include "../config.h"
```

Comment:
There is no point in it. My vim syntax and error checker mocks up, that it can' find `config.h` and the symbols included in it.

I had issues giving additional compiler flags (like `-I.`), so I used it as a quick soultion. I have to go forward and fix the actual issues. Sorry for the noise.